tests/common/fixtures/duthost_utils.py

```diff
@@ -225,3 +226,79 @@ def vlan_ports_list(duthosts, rand_one_dut_hostname, rand_selected_dut, tbinfo,
             vlan_ports_list.append(vlan_port)
 
     return vlan_ports_list
+
+def compare_network(src_ipprefix, dst_ipprefix):
+    src_network = ipaddress.IPv4Interface(src_ipprefix).network
+    dst_network = ipaddress.IPv4Interface(dst_ipprefix).network
+    return src_network.overlaps(dst_network)
+
+@pytest.fixture(scope="module")
+def utils_vlan_intfs_dict_orig(duthosts, rand_one_dut_hostname, tbinfo):
```

Comment:
>utils_vlan_intfs_dict_orig [](http://example.com/codeflow?start=4&length=26)

It's not easy to understand the function just from the name. They are implemented to be reused. Could you add some function level comments, explaining the purpose, parameter, and return values.

Reply:
Also applicable to other new functions in this file.

Reply:
Added